libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -238,6 +238,26 @@ class AC_AttitudeControl {
     // sanity check parameters.  should be called once before take-off
     virtual void parameter_sanity_check() {}
 
+    // return true if the rpy mix is at lowest value
+    virtual bool is_throttle_mix_min() const { return true; }
+
+    // control rpy throttle mix
+    virtual void set_throttle_mix_min() {}
+    virtual void set_throttle_mix_mid() {}
+    virtual void set_throttle_mix_max() {}
+
+    // enable use of flybass passthrough on heli
+    virtual void use_flybar_passthrough(bool passthrough, bool tail_passthrough) {}
+
+	// use_leaky_i - controls whether we use leaky i term for body-frame to motor output stage on heli
+	virtual void use_leaky_i(bool leaky_i) {}
```

Comment:
I'm not sure why it's necessary to expose these because they should only ever be called when the frame is a helicopter..